include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -299,7 +299,7 @@
 
 // When compile with Intel(R) oneAPI DPC++ Compiler macro is on
 // We need this macro to use compiler specific implementation of SYCL
```

Comment:
Let's remove the comment. The first line is not relevant anymore, the second one is not required due to the self-descriptive name of the macro (`SYCL_IMPLEMENTATION_INTEL`).

```suggestion
```

Reply:
Some from internal talks I decided that using the `SYCL_IMPLEMENTATION_INTEL` macro directly made more sense then changing this. Given this change, would you still like me to remove this comment?

Reply:
Thanks, it is even better. With that change, I would recommend removing the comment and `_ONEDPL_SYCL_INTEL_COMPILER` macro definition altogether: this macro will have applications.

Reply:
Sounds good. Iv removed the macro and the comment